src/misc/credentials/CredentialsProvider.ts

```diff
@@ -66,10 +66,20 @@ export class CredentialsProvider {
 	/**
 	 * Returns the stored credentials infos of all internal users, i.e. users that have a "real" tutanota account and not the ones that
 	 * have a secure external mailbox.
+	 *
+	 * @param opts.sorted If true, sort the return value by login. Use this if you intend to display this as a list to the user.
 	 */
-	async getInternalCredentialsInfos(): Promise<ReadonlyArray<CredentialsInfo>> {
+	async getInternalCredentialsInfos(opts: { sorted: boolean } = { sorted: false }): Promise<ReadonlyArray<CredentialsInfo>> {
```

Comment:
I don't see a reason to add it as a parameter, we should just always do it, either here or in ViewModel, whatever is more appropriate